x/migration/module/autocli.go

```diff
@@ -50,6 +50,15 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "morseAccountState"}},
 					Skip:           true, // skipped because authority gated
 				},
+				{
+					RpcMethod:      "ClaimMorseAccount",
+					Use:            "claim-morse-account [morse-src-address-hex] [morse-signature-hex]",
+					Short:          "Claim the account balance of the given Morse account address",
+					Long:           "Claim the account balance of the given Morse account address, by signing the message with the private key of the Morse account.",
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "morse_src_address"}, {ProtoField: "morse_signature"}},
+					Skip:           true, // skipped because autoCLI cannot handle signing
+					// TODO_UPNEXT(@bryanchriswhite#1034): Add morse account claiming CLI.
```

Comment:
Can you either:
1. #PUC for TODO to add concrete copy-pasta examples
2. Add examples in this PR similar to `x/supplier/module/autocli.go`